### ISSUE TITLE
Webpack exists at start with code 0

### ISSUE THREAD

**Issue by JrmKrb** [shakacode/react_on_rails#553]
Hi,

It's been a few days I upgraded to Rails 5.0.0.1, and around the same time, "foreman -f Procfile.dev" started to exit with code 0 automatically after creating vendor and app bundles. I'm not sure it's linked with Rails version but I can't find a way to fix it. Here are the logs of a typical foreman run.

`20:01:13 client.1 | Hash: e493a6d9c109c6144672
20:01:13 client.1 | Version: webpack 1.13.2
20:01:13 client.1 | Time: 9284ms
20:01:13 client.1 |            Asset     Size  Chunks             Chunk Names
20:01:13 client.1 |    app-bundle.js  9.88 MB       0  [emitted]  app
20:01:13 client.1 | vendor-bundle.js  1.85 MB       1  [emitted]  vendor
20:01:13 client.1 |    [0] multi app 484 bytes {0} [built]
20:01:13 client.1 |    [0] multi vendor 76 bytes {1} [built]
20:01:13 client.1 |     + 1024 hidden modules
20:01:14 client.1 | exited with code 0
20:01:14 system   | sending SIGTERM to all processes
20:01:14 web.1    | [8874] - Gracefully shutting down workers...
20:01:14 web.1    | [8874] === puma shutdown: 2016-09-20 20:01:14 +0200 ===
20:01:14 web.1    | [8874] - Goodbye!
20:01:14 web.1    | Exiting
20:01:15 web.1    | exited with code 0
`

After rebooting several times, it happened once that it could launch properly, but webpack wasn't listening to changes made to my reactjs code and I had to launch it again every time.

Did I miss something or has anyone faced this issue before ? I'm running Ubuntu 16.04 if it helps.

Thanks for your help !
Jeremy.

**Comment by justin808** [comment]
@JrmKrb It sounds like you have some processes that didn't get killed when you restarted. That happens all the time with foreman.

Try this: https://github.com/justin808/justin808-dotfiles/blob/master/zsh/rails.zsh#L37

**Comment by JrmKrb** [comment]
Thanks for your answer @justin808 !

I tried it but it didn't change anything. I don't think it comes from here because even after rebooting the issue is still here whereas the so-called processes shouldn't be here anymore. I can't even find a pattern of when it exits automatically like here or when it works but without listening properly to changes made to client/ folder. I tried to uninstall npm/node-js/foreman and start again but it didn't work.
